src/libraries/System.Speech/pkg/System.Speech.pkgproj

```diff
@@ -8,9 +8,10 @@
     <InboxOnTargetFramework Include="net45">
       <AsFrameworkReference>true</AsFrameworkReference>
     </InboxOnTargetFramework>
-    <PackageFile Include=".\build\System.Speech.targets">
-      <TargetPath>build\netcoreapp2.0\</TargetPath>
-    </PackageFile>
+    <PackageFile Include="buildTransitive\System.Speech.targets"
+                 TargetPath="buildTransitive\netcoreapp2.0" />
```

Comment:
Consider duplicating as `build` folder as well if you need this to work in a NuGet client version that doesn’t understand `buildTransitive`

Reply:
I'm fine with not supporting that scenario...

Reply:
Something to keep in mind moving forward.  Packages.config never honors it so you'll definitely want to duplicate if you're intending to apply those to a .NETFramework project.  The [feature](https://github.com/NuGet/NuGet.Client/commit/8d300225298df9c6af6c784efffa7522109f0096) was introduced in NuGet 5.0.0.5800 so some older SDKs won't have it.

Reply:
Thanks for the info. I will keep this in mind in case we want to upgrade .NET Framework versions as well at some point.